packages/playground/remote/src/lib/worker-thread.ts

```diff
@@ -189,6 +191,47 @@ export class PlaygroundWorkerEndpoint extends PHPWorker {
 	}
 }
 
+async function downloadWordPressAssets(php: PHP) {
+	const wpVersion = await getWordPressVersionFromPhp(php);
+	fetch(`${wordPressSiteUrl}/wp-${wpVersion}/wordpress-static.zip`)
```

Comment:
Just a heads up. There is now a function codifying the static file directory naming of `wp-{version}`:
https://github.com/WordPress/wordpress-playground/blob/4c8b417b3024db671eeac4258255ce8eff739b7f/packages/playground/wordpress-builds/src/index.ts#L14

Reply:
Thanks, updated it in [4a1f8d0](https://github.com/WordPress/wordpress-playground/pull/1532/commits/4a1f8d018de633a9dfb5102e553f3ed93339b37f)